Data/Hashable/Class.hs

```diff
@@ -699,6 +699,13 @@ instance Hashable (Type.Reflection.TypeRep a) where
     {-# INLINE hash #-}
 #endif
 
+#if __GLASGOW_HASKELL__ >= 710
+instance Hashable Fingerprint where
```

Comment:
Why only for GHC 7.10 and later?

Reply:
Whoops. I've changed this to go all the way back to 7.2.